pydra/engine/task.py

```diff
@@ -519,10 +518,19 @@ def _run_task(self):
             values = execute(args, strip=self.strip)
             self.output_ = dict(zip(keys, values))
             if self.output_["return_code"]:
+                msg = f"Error running '{self.name}' task with {args}:"
                 if self.output_["stderr"]:
-                    raise RuntimeError(self.output_["stderr"])
-                else:
-                    raise RuntimeError(self.output_["stdout"])
+                    msg += "\n\nstderr:\n" + self.output_["stderr"]
+                if self.output_["stdout"]:
+                    msg += "\n\nstderr:\n" + self.output_["stdout"]
+                raise RuntimeError(msg)
+
+    def _join_cmd_args(self, args):
```

Comment:
Not 100% sure about this, feels a bit hacky just skipping the executable but it should work. Any suggestions on a more elegant solution?